string controller_name
float64 duration_microsecs
